composer.json

```diff
@@ -1,10 +1,55 @@
 {
-    "name"       : "automattic/msm-sitemap",
-    "description": "Comprehensive sitemaps for your WordPress.com VIP site.",
-    "homepage"   : "https://vip.wordpress.com/plugins/msm-sitemap/",
-    "type"       : "wordpress-plugin",
-    "license"    : "GPL-2.0+",
-    "require"    : {
-        "composer/installers": "~1.0"
+  "name": "automattic/msm-sitemap",
+  "description": "Comprehensive sitemaps for your WordPress.com VIP site.",
+  "homepage": "https://github.com/Automattic/msm-sitemap/",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0+",
+  "authors": [
+    {
+      "name": "Automattic",
+      "homepage": "http://automattic.com/"
     }
+  ],
+  "support": {
+    "issues": "https://github.com/Automattic/msm-sitemap/issues",
+    "source": "https://github.com/Automattic/msm-sitemap"
+  },
+  "require": {
+    "composer/installers": "~1.0",
+    "php": ">=5.6"
+  },
+  "require-dev": {
+    "automattic/vipwpcs": "^2.2",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7",
+    "php-parallel-lint/php-parallel-lint": "^1.0",
+    "phpcompatibility/phpcompatibility-wp": "^2.1",
+    "phpunit/phpunit": "^4 || ^5 || ^6 || ^7",
+    "squizlabs/php_codesniffer": "^3.5",
+    "wp-coding-standards/wpcs": "^2.3.0",
+    "yoast/phpunit-polyfills": "^0.2.0"
+  },
+  "scripts": {
+    "cs": [
+      "@php ./vendor/bin/phpcs -p -s -v -n . --standard=\"WordPress-VIP-Go\" --extensions=php --ignore=\"/vendor/*,/node_modules/*,/tests/*\""
```

Comment:
I would highly recommend moving this configuration into a `.phpcs.xml.dist` file, since other exclusions can configurations (like setting what the text-domain should be, or adding PHPCompatibilityWP) can also be done there. [Example for wp-parsely](https://github.com/Parsely/wp-parsely/blob/develop/.phpcs.xml.dist).

Then the script simply becomes:
```json
"cs": [
	"@php ./vendor/bin/phpcs"
],
```

...which means anyone who runs `phpcs` instead will get a consistent result.